src/Utils/CMakeLists.txt

```
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.16)
+
+project(Utils
+	LANGUAGES CXX
+	VERSION 0.0.1)
+
+qt_standard_project_setup()
+
+set(CMAKE_AUTOUIC ON)
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+find_package(QT NAMES Qt6 Qt5 REQUIRED COMPONENTS Core)
```

Da das Projekt Qt6 only ist kann man Qt5 rausschmeissen.
```suggestion
find_package(QT NAMES Qt6 REQUIRED COMPONENTS Core)
```

---

4cf884b